config/settings_data.json

```diff
@@ -9,7 +9,7 @@
           "announcement-bar-0": {
             "type": "announcement",
             "settings": {
-              "text": "Welcome to our store",
+              "heading": "Welcome to our store",
```

Comment:
Sorry Melissa, I saved in the editor, and it overwrote this file 🤦 could you revert back this file?
If not, I can take care of it.

Reply:
Amazing!